webapp/app/forms/steps/lotse_multistep_flow_steps/confirmation_steps.py

```diff
@@ -60,24 +61,39 @@ class StepFiling(DisplayStep):
     name = 'filing'
 
     def __init__(self, **kwargs):
-        super(StepFiling, self).__init__(title=_('form.lotse.filing.title'),
-                                         intro=_('form.lotse.filing.intro'),
+        super(StepFiling, self).__init__(title=_('form.lotse.filing.success.header-title'),
                                          **kwargs)
 
     def render(self, data, render_info):
-        render_info.additional_info['disable_extended_footer'] = True
         if render_info.additional_info['elster_data']['was_successful']:
-            return render_template('lotse/display_filing_success.html', render_info=render_info,
-                                   elster_data=render_info.additional_info['elster_data'],
-                                   header_title=_('form.lotse.filing.header-title'),
-                                   tax_number_provided=data.get('steuernummer_exists') == 'yes'
-                                   if data.get('steuernummer_exists')
-                                   else None)
+            props_dict = FilingSuccessProps(
+                next_url=render_info.next_url,
+                transferTicket=render_info.additional_info['elster_data']['transfer_ticket'],
+                downloadUrl=url_for("download_pdf"),
+                taxNumberProvided=data.get('steuernummer_exists') == 'yes'
+                if data.get('steuernummer_exists')
```

Comment:
This should be:

``'yes' if data.get('steuernummer_exists') == 'yes' else None``

steuernummer_exists could also be 'no' or?

Reply:
Maybe: ``taxNumberProvided=data.get('steuernummer_exists')=='yes``?

Reply:
I just put it the way it was before. We need a boolean value but only if that attribute is within data right?

Reply:
Is within the data and the value of steuernummer_exists is 'yes' it also can be 'no'

Reply:
Your short proposal `taxNumberProvided=data.get('steuernummer_exists')=='yes'` shoudl be fine! What do you think @JulianRoesner ?

Reply:
So, the behavior so far was:
- data['steuernummer_exists'] = 'yes' -> taxNumberProvided = True
- data['steuernummer_exists'] = 'no' -> taxNumberProvided = False
- 'steuernummer_exists' not in data -> taxNumberProvided = None

The proposed changes would lead to:
- data['steuernummer_exists'] = 'yes' -> taxNumberProvided = True
- data['steuernummer_exists'] = 'no' -> taxNumberProvided = False
- 'steuernummer_exists' not in data -> taxNumberProvided = False

This could potentially lead to incorrect data in Plausible (more cases without Steuernummer which are just incorrect data). This should only happen, if someone goes directly to the filing page with incorrect data and notwithstanding through the actual flow. However, this is why we tested for None before

Reply:
Okay than it's fine for me :)

Reply:
I agree about the one liner and maybe add a comment pls for our future selfs that they are not getting confused about this :)

Reply:
Wish granted! :)